src/model/input.ts

```diff
@@ -14,64 +14,66 @@ const core = require('@actions/core');
  * Todo: rename to UserInput and remove anything that is not direct input from the user / ci workflow
  */
 class Input {
-  public static cliOptions;
   public static githubInputEnabled: boolean = true;
 
-  // also enabled debug logging for cloud runner
-  static get cloudRunnerTests(): boolean {
-    return Input.getInput(`cloudRunnerTests`) || Input.getInput(`CloudRunnerTests`) || false;
-  }
+  public static getInput(query) {
+    if (Input.githubInputEnabled) {
+      const coreInput = core.getInput(query);
+      if (coreInput && coreInput !== '') {
+        return coreInput;
+      }
+    }
+    const alternativeQuery = Input.ToEnvVarFormat(query);
 
-  private static getInput(query) {
-    const coreInput = core.getInput(query);
-    if (Input.githubInputEnabled && coreInput && coreInput !== '') {
-      return coreInput;
+    // query input sources
+    if (CLI.query(query, alternativeQuery)) {
+      return CLI.query(query, alternativeQuery);
+    }
+    if (CloudRunnerQueryOverride.query(query, alternativeQuery)) {
+      return CloudRunnerQueryOverride.query(query, alternativeQuery);
+    }
+    if (process.env[query] !== undefined) {
+      return process.env[query];
+    }
+    if (alternativeQuery !== query && process.env[alternativeQuery] !== undefined) {
+      return process.env[alternativeQuery];
     }
```

Comment:
Please use empty lines between if statements unless they're one-liners. Optionally with a comment above them (or ideally they'd be self-descriptive because they'd call methods from the domain model classes.